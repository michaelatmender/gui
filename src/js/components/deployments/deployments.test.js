import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { createMount } from '@material-ui/core/test-utils';
import Deployments from './deployments';

it('renders correctly', () => {
  const context = {
    childContextTypes: {
      router: () => {}
    },
    context: {
      router: {
        route: {
          location: {
            hash: '',
            pathname: '',
            search: '',
            state: ''
          },
          match: { params: {}, isExact: false, path: '', url: '' }
        }
      }
    }
  };
  const tree = createMount()(
    <MemoryRouter>
      <Deployments location={{ search: '' }} />
    </MemoryRouter>,
    context
  );
  expect(tree.html()).toMatchSnapshot();
});
