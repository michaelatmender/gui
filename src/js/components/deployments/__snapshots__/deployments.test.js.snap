// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"relative\\"><button class=\\"MuiButtonBase-root MuiButton-root top-right-button MuiButton-contained MuiButton-containedSecondary\\" tabindex=\\"0\\" type=\\"button\\" style=\\"position: absolute;\\"><span class=\\"MuiButton-label\\">Create a deployment</span><span class=\\"MuiTouchRipple-root\\"></span></button><div class=\\"MuiTabs-root\\" style=\\"display: inline-block;\\"><div class=\\"MuiTabs-scroller MuiTabs-fixed\\" style=\\"overflow: hidden;\\"><div class=\\"MuiTabs-flexContainer\\" role=\\"tablist\\"><a class=\\"MuiButtonBase-root MuiTab-root MuiTab-textColorInherit Mui-selected\\" tabindex=\\"0\\" role=\\"tab\\" aria-disabled=\\"false\\" aria-selected=\\"true\\" href=\\"/deployments/active\\"><span class=\\"MuiTab-wrapper\\">Active</span><span class=\\"MuiTouchRipple-root\\"></span></a><a class=\\"MuiButtonBase-root MuiTab-root MuiTab-textColorInherit\\" tabindex=\\"0\\" role=\\"tab\\" aria-disabled=\\"false\\" aria-selected=\\"false\\" href=\\"/deployments/finished\\"><span class=\\"MuiTab-wrapper\\">Finished</span><span class=\\"MuiTouchRipple-root\\"></span></a></div><span class=\\"PrivateTabIndicator-root-42 PrivateTabIndicator-colorSecondary-44 MuiTabs-indicator\\" style=\\"left: 0px; width: 0px;\\"></span></div></div><div class=\\"margin-top\\"><div class=\\"fadeIn\\"><div class=\\"deploy-table-contain\\"><div class=\\"loaderContainer\\"><div class=\\"loader\\"><span class=\\"dot dot_1\\"></span><span class=\\"dot dot_2\\"></span><span class=\\"dot dot_3\\"></span><span class=\\"dot dot_4\\"></span></div></div><div class=\\"hidden\\"><p>Pending and ongoing deployments will appear here. </p><p><a>Create a deployment</a> to get started</p><img src=\\"assets/img/deployments.png\\" alt=\\"In progress\\"></div></div></div></div></div>"`;
